tests/fixtures/bugs.json

```diff
@@ -31,4 +31,4 @@
 us from reaching the mPromiseGripArray.RemoveElement() call, and then the\n> lack of cycle collection could be causing us to leak...\n\nGiven that push, this is indeed the most plausible reason.  I'll write a patch.", "id": 11947100, "creation_time": "2016-12-29T19:44:48Z", "author": "ehsan@mozilla.com"}, {"count": 9, "text": "Andrew, what's the process for testing a patch for one of these bugs?", "id": 11947101, "creation_time": "2016-12-29T19:45:12Z", "author": "ehsan@mozilla.com"}, {"count": 10, "text": "Probably back out https://hg.mozilla.org/mozilla-central/rev/90dd139d7578 and see if things fail with/without the patch?", "id": 11947120, "creation_time": "2016-12-29T19:51:26Z", "author": "bzbarsky@mit.edu"}, {"count": 11, "text": "Created attachment 8822500\nEnsure that all AudioContext members that need to participate in cycle collection do so", "id": 11947126, "creation_time": "2016-12-29T19:54:16Z", "author": "ehsan@mozilla.com"}, {"count": 12, "text": "Comment on attachment 8822500\nEnsure that all AudioContext members that need to participate in cycle collection do so\n\nr=me but I have low confidence that I'd catch issues like someone on another thread having a ref to something in mPromiseGripArray (its ostensible job is to enable that, right?) and us pulling the rug out from under it.  At the very least, it's worth a comment for mPromiseGripArray explaining what the ownership model is and why clearing it is safe....", "id": 11947129, "creation_time": "2016-12-29T19:56:41Z", "author": "bzbarsky@mit.edu"}, {"count": 13, "text": "Comment on attachment 8822500\nEnsure that all AudioContext members that need to participate in cycle collection do so\n\nWhen we transfer the promise to another thread, we cast it to void* and then cast it back to Promise* when we receive it again on the main thread.  But I'll ask another review from karlt.", "id": 11947159, "creation_time": "2016-12-29T20:05:59Z", "author": "ehsan@mozilla.com"}, {"count": 14, "text": "https://treeherder.mozilla.org/#/jobs?repo=try&revision=608191d559885134ede4dfdb8fbe082b2f481128", "id": 11947160, "creation_time": "2016-12-29T20:06:10Z", "author": "ehsan@mozilla.com"}, {"count": 15, "text": "Yes, backout https://hg.mozilla.org/mozilla-central/rev/90dd139d7578 to test it. This is also the only leak I've seen in reftests, so you can also push the backout along with this patch (or just land this and I'll do it later).", "id": 11947192, "creation_time": "2016-12-29T20:07:44Z", "author": "ahal@mozilla.com"}, {"count": 16, "text": "(In reply to :Ehsan Akhgari from comment #14)\n> https://treeherder.mozilla.org/#/\n> jobs?repo=try&revision=608191d559885134ede4dfdb8fbe082b2f481128\n\nThe patch fixes the leak! \\o/", "id": 11947354, "creation_time": "2016-12-29T21:30:47Z", "author": "ehsan@mozilla.com"}, {"count": 17, "text": "Comment on attachment 8822500\nEnsure that all AudioContext members that need to participate in cycle collection do so\n\nAsyncDecodeWebAudio creates a MediaDe
```

Comment:
Could you remove the changes from this file altogether?

Reply:
To remove the file, you can simply do:
```
git checkout test-for-assignee
git reset HEAD~1
git checkout -- tests/fixtures/bugs.json
git add tests/test_assignee.py
git commit -m "Add basic test for the Assignee model"
git push --force
```

Reply:
hm... I'm not sure I did corretly

Reply:
No, it's still there :)

Reply:
Here's another way to do it:
1. Clone this repository in another directory
2. Copy the tests/fixtures/bugs.json from the newly cloned repository to your repository
3. git add tests/fixtures/bugs.json
4. git commit -m "tmp"
5. git rebase -i HEAD~2
6. Select to squash the second commit into the first
7. git push --force origin test-for-assignee

Reply:
I did the above. Can you check it?

Reply:
As you can see, the file is still there :)

Reply:
You want to erase the fixtures/bugs.json file?

Reply:
No, I want the PR not to make any changes to the file.

Reply:
Oh...I see....

Reply:
Let's try yet another thing:
```
cp tests/test_assignee.py tmp.py
git checkout master
git branch -D test-for-assignee
git checkout -b test-for-assignee
mv tmp.py tests/test_assignee.py
git add tests/test_assignee.py
git commit -m "Add basic test for the Assignee model"
git push origin test-for-assignee --force
```

Reply:
Ok, I did it. Thanks for help